sparse/rest/server.go

```diff
@@ -9,6 +9,15 @@ import (
 	"github.com/longhorn/sparse-tools/sparse"
 )
 
+type DataSyncServer interface {
```

Comment:
This is good! 

However, because this interface is only used internally (rest pkg), so suggest renaming all methods to lower camel case.